src/new/templates/rust/ui/hyperapp/skeleton-app/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[dependencies]
+anyhow = "1.0"
+process_macros = "0.1"
+serde_json = "1.0"
+wit-bindgen = "0.42.1"
+
+[dependencies.hyperapp_skeleton_caller_utils]
+optional = true
+path = "../target/hyperapp-skeleton-caller-utils"
+
+[dependencies.hyperprocess_macro]
+git = "https://github.com/hyperware-ai/hyperprocess-macro"
+rev = "aaf7e02"
+
+[dependencies.hyperware_process_lib]
+features = ["hyperapp"]
+git = "https://github.com/hyperware-ai/process_lib"
+rev = "e8b065179ce5d15893a23142416e59c87e0f31f6"
+
+[dependencies.serde]
+features = ["derive"]
+version = "1.0"
+
+[features]
+caller-utils = ["hyperapp_skeleton_caller_utils"]
+default = ["caller-utils"]
```

Comment:
I think we should remove this because it will interfere with fresh builds. We should test to confirm/falsify: does this mess up building fresh?

Reply:
Again, doesn’t make a diff with fresh builds. Remove or keep, your choice.

Reply:
Done.